Documentation/getting-started.md

```diff
@@ -0,0 +1,128 @@
+---
+weight: 201
+toc: true
+title: Getting Started
+menu:
+    docs:
+        parent: user-guides
+lead: ""
+images: []
+draft: false
+description: Getting started page for Platform Guide
```

Comment:
I don't recall where `description` is used. Is this shown somewhere in the website?

Reply:
AFAIK, adding a description for each page helps with SEO. Maybe we can modify description of each of the pages in a PR if need be.